presto-hive/src/test/java/com/facebook/presto/hive/statistics/TestQuickStatsProvider.java

```diff
@@ -336,10 +339,10 @@ else if (partitionStatistics2.equals(empty())) {
 
     @Test
     public void quickStatsBuildTimeIsBounded()
-            throws InterruptedException
+            throws Exception
     {
         QuickStatsBuilder longRunningQuickStatsBuilderMock = mock(QuickStatsBuilder.class);
-        ImmutableMap<String, Integer> mockPerPartitionStatsFetchTimes = ImmutableMap.of("p1", 10, "p2", 20, "p3", 500, "p4", 800);
+        ImmutableMap<String, Integer> mockPerPartitionStatsFetchTimes = ImmutableMap.of("p1", 10, "p2", 20, "p3", 1500, "p4", 1800);
```

Comment:
This was not causing test failures - but I increased these background sleep times as a precaution.